Gopkg.lock

```diff
@@ -199,12 +199,6 @@
   revision = "bfe3c2e8f406bf352bc8df81f98c752224867349"
   version = "v2.17.11"
 
-[[projects]]
-  branch = "master"
-  name = "github.com/shirou/w32"
```

Comment:
I've noticed this sneaking into my lock file as well when I do `dep ensure`. I don't think it's good practice to edit Gopkg.lock directly though - we need to figure out where this is coming from and either keep it or make sure it doesn't keep coming back. It may be that it's a necessary transitive dependency.

Reply:
That package is required if you do a windows build of the `github.com/shirou/gopsutil` package.
```
// +build windows

package process
```
(It's also why `cpu` and `net` are included in the package list: `packages = ["cpu","host","internal/common","mem","net","process"]`)

`dep status` for that package gives me this:
```
PROJECT                                  CONSTRAINT     VERSION        REVISION  LATEST   PKGS USED
github.com/shirou/gopsutil               ^2.17.11       v2.17.11       bfe3c2e   v2.18.01  4
```

I tried various things in an attempt to reproduce this to have `github.com/shirou/w32` added to my `Gopkg.lock`, but the only thing that worked was `dep ensure -update` or just completely rebuilding the `Gopkg.*` files altogether. I even tried various versions of `dep`. So I guess in summary I'm not sure how you guys are getting *only* this project added to the lock file.

Reply:
I confess, i cheated and edited the lock file to make it go away